db/migration/V003__Queue_Data.sql

```diff
@@ -15,3 +15,5 @@ INSERT INTO queue (id,subject_id,encrypted_resp,date_sent,date_received) VALUES
 INSERT INTO queue (id,subject_id,encrypted_resp,date_sent,date_received) VALUES ('6bae8ab6-bb00-4215-b34a-d1e6a8f3ea34','7bfc3b07-a97d-4e11-8ac6-b970c1745476','MIICjQYJKoZIhvcNAQcDoIICfjCCAnoCAQAxggHiMIIB3gIBADCBxTCBrDELMAkGA1UEBhMCREUxDDAKBgNVBAgMA1JMUDEOMAwGA1UEBwwFTWFpbnoxETAPBgNVBAoMCFVNIE1haW56MR4wHAYDVQQLDBVQcmV2ZW50aXZlIENhcmRpb2xvZ3kxFTATBgNVBAMMDEdDU1N0dWR5Q2VydDE1MDMGCSqGSIb3DQEJARYmc3Zlbi1vbGl2ZXIudHJvZWJzQHVuaW1lZGl6aW4tbWFpbnouZGUCFHNImNekULpbsaUS+4bJqvxUWlrdMA0GCSqGSIb3DQEBAQUABIIBAIzwcMtFWOkLMqEvlSU+JrYaTtBTZ6D2AqtZryF2T2cktiUcvGcqZ9B8/EDpV6gEy7fCoA60kJoyb3/XIQkruwF5KQrm4WA2ttuTNlJf2n3H4oF4BnRpkPS93D/A2KariLPxC+9VZBaRO8vhKh0g3NoDpnP6LZIQ8fhCPKt6rwghRrMBMEHf9fK9L0SLaIuWgKkQJfaEwzSdEJOrn1w0yNgDlbkb1igBWSybfIE4ZLzV35KCRvdxLDOW16DuR5tuhYu1Itd5m5S/ej4zW6UHmcnBSmpAWIHZuZgSZBqixlL6wW2bj3i+juZntVZFFHnyZ/yvt1hAYuNgiTMh8r2r/mwwgY4GCSqGSIb3DQEHATAdBglghkgBZQMEASoEEPrUg+bC8Yf5rIc1X7Fi3YigYgRgiaXhQVJUr//Fqe/0my01ICwE7HDstHSLcQx2tBe0xrBjXsjvFDMVESYapnbyjCzwU7IkhFqnnjokAWHb9nCPj/C/pVk50ILnGdXQjY8XT9Mr/58iunTbNAVZBlmrKglR','2021-01-27 10:45:00.945','2021-01-25 10:45:00.948');
 INSERT INTO queue (id,subject_id,encrypted_resp,date_sent,date_received) VALUES ('89c51691-694e-4106-bf71-39b8c1e99511','7bfc3b07-a97d-4e11-8ac6-b970c1745476','MIICjQYJKoZIhvcNAQcDoIICfjCCAnoCAQAxggHiMIIB3gIBADCBxTCBrDELMAkGA1UEBhMCREUxDDAKBgNVBAgMA1JMUDEOMAwGA1UEBwwFTWFpbnoxETAPBgNVBAoMCFVNIE1haW56MR4wHAYDVQQLDBVQcmV2ZW50aXZlIENhcmRpb2xvZ3kxFTATBgNVBAMMDEdDU1N0dWR5Q2VydDE1MDMGCSqGSIb3DQEJARYmc3Zlbi1vbGl2ZXIudHJvZWJzQHVuaW1lZGl6aW4tbWFpbnouZGUCFHNImNekULpbsaUS+4bJqvxUWlrdMA0GCSqGSIb3DQEBAQUABIIBAFFb9tx8aOifXyIA41MNFYI+aBSm+8HL4ZYfVMa+XZqU1ajnSu/NbmNX0yjoWkYEElHctAi35Vs2he+GJh9qBYtnRTGbnel3VsfdM4LjqK3GrPmbffeUOQRJ3JRMEeUAJJ4STUxiz15OLyzhdtt4P6wlcZpBBcOA4ZsFOm3gzmbwuGv9qyecNQeSFr42g0rAV714o1K1MyqJ0mRAbskYuwwcd2UtyTR19H8LFBqOb+aq50WEjYbZOo1itpMoIaqzvqdHwFiKLBrY5xxk+ng1EfAwHkZaaMIvmsF4TaYVhzTbVKCSHwEI5QmxqaTyxBvKYivo8w0Q8fXsi2kvwIDJbhkwgY4GCSqGSIb3DQEHATAdBglghkgBZQMEASoEEPMjYPKZtUc0om9MeoP1KrqgYgRgNhpKGMvVPH6P0hnNiVTC4XsBZl0RV/GQAMR+msjrAm0JwahgaVO7njCttNw+TGtXpPfzm0m0bNiqQxJz7RSymXxD9suokEBElgoK+vbDvmTETpIxNWOFTrl0L5XVuAYo','2021-02-01 09:17:16.422','2021-01-30 09:17:16.422');
+ALTER TABLE queue ADD questionnaire_id varchar(255);
+ALTER TABLE queue ADD version varchar;
```

Comment:
My personal opinion:

either create a new migration for those changes to prevent conflicts in (theoretically) existing projects
or (when we don't care about possible conflicts) directly change the creation of the 'queue' table

Reply:
agree!